petclinic.yml

```diff
@@ -9,12 +10,18 @@ application:
       application-pic: bind(haproxy#app.petclinic-image)
   configuration:
     configuration.clusterSize: 1
+    configuration.haproxy-docker-image-id: "qubell/haproxy:latest"
+    configuration.database-initializer-docker-image-id: "tutum/mysql"
```

Comment:
I'm not sure if it is a good idea to make these images configurable. The manifest is unlikely to work with any other image.